pkg/virt-handler/cmd-client/client_test.go

```diff
@@ -228,5 +228,83 @@ var _ = Describe("Virt remote commands", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 		})
+
+		Context("GetGuestInfo", func() {
+			var (
+				mockCmdClient *cmdv1.MockCmdClient
+				client        LauncherClient
+			)
+
+			BeforeEach(func() {
+				ctrl := gomock.NewController(GinkgoT())
+				mockCmdClient = cmdv1.NewMockCmdClient(ctrl)
+				client = newV1Client(mockCmdClient, nil)
+			})
+
+			var (
+				expectGetGuestInfo = func() *gomock.Call {
+					return mockCmdClient.EXPECT().GetGuestInfo(gomock.Any(), &cmdv1.EmptyRequest{})
+				}
+			)
```

Comment:
This definition looks a bit weird. Why not make it a direct assignment?

Reply:
Not sure what you mean here? From what I know this is also how we do it in other tests.

Reply:
```
			expectGetGuestInfo := func() *gomock.Call {
				return mockCmdClient.EXPECT().GetGuestInfo(gomock.Any(), &cmdv1.EmptyRequest{})
			}
```

Looks better to me, but essentially it's a nit